sdk/communication/Azure.Communication.CallingServer/tests/SessionRecords/Sample1_CallClient/CreateCallAsyncAsync.json

```diff
@@ -0,0 +1,61 @@
+{
+  "Entries": [
+    {
+      "RequestUri": "https://acstestbot3.communication.azure.com/calling/calls?api-version=2021-04-15-preview1",
+      "RequestMethod": "POST",
+      "RequestHeaders": {
+        "Accept": "application/json",
+        "Authorization": "Sanitized",
+        "Content-Length": "354",
+        "Content-Type": "application/json",
+        "Date": "Wed, 02 Jun 2021 21:28:29 GMT",
+        "traceparent": "00-f03b3c0995adc24882d0cb3f6d4ef016-32e1ef3fa4d5b94d-00",
+        "User-Agent": "azsdk-net-Communication.CallingServer/1.0.0-alpha.20210602.1 (.NET Framework 4.8.4300.0; Microsoft Windows 10.0.19043 )",
+        "x-ms-client-request-id": "750c6da09b80fc970cd7aa322fd6b1c9",
+        "x-ms-content-sha256": "Sanitized",
+        "x-ms-return-client-request-id": "true"
+      },
+      "RequestBody": {
+        "targets": [
+          {
+            "phoneNumber": {
+              "value": "\u002B15129104698"
+            }
+          }
+        ],
+        "source": {
+          "communicationUser": {
+            "id": "8:acs:02665c56-277e-4c59-bab4-c475caa3ee80_0000000a-6e82-5627-54b7-a43a0d00395f"
+          }
+        },
+        "callbackUri": "https://dummy.ngrok.io/api/incident/callback?SecretKey=helloworld",
+        "requestedModalities": [
+          "audio"
+        ],
+        "requestedCallEvents": [
+          "participantsUpdated",
+          "dtmfReceived"
+        ]
+      },
+      "StatusCode": 201,
+      "ResponseHeaders": {
+        "Client-Request-Id": "77074750-189c-4615-843f-b357d68fa3b5",
+        "Content-Length": "52",
+        "Content-Type": "application/json; charset=utf-8",
+        "Date": "Wed, 02 Jun 2021 21:28:28 GMT",
+        "X-Azure-Ref": "0/Pe3YAAAAADZL\u002B\u002BDA03YQangI05vTMI1V1NURURHRTA4MTQAOWZjN2I1MTktYThjYy00Zjg5LTkzNWUtYzkxNDhhZTA5ZTgx",
+        "X-Cache": "CONFIG_NOCACHE",
+        "X-Microsoft-Skype-Chain-ID": "80b47d16-ed1f-4c8b-88b9-3f6d1de9b3f5"
+      },
+      "ResponseBody": {
+        "callLegId": "6c1f6900-5681-4271-9cf6-f5c1a3a64bc3"
+      }
+    }
+  ],
+  "Variables": {
+    "AZURE_PHONE_NUMBER": "\u002B15129104698",
```

Comment:
We should be redacting our phone numbers in the recordings. See SMS or Phone Numbers SDK for an example of how to do this.

Reply:
https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/communication/Azure.Communication.Sms/tests/SmsClientRecordedTestSanitizer.cs

Reply:
This is addressed in updates.